contracts/AKAP.sol

```diff
@@ -26,6 +26,9 @@ contract AKAP is ERC721Full {
         bytes nodeBody;
     }
 
+    event NodeClaim(address indexed _sender, uint256 indexed _nodeId, uint _parentId, bytes _label);
```

Comment:
I think it would make sense to have parentId also be indexed, so that I can easily see when new children are created.

Reply:
Yes, we can add indexed attribute up to 3 parameters.